metaworld/envs/mujoco/sawyer_xyz/v2/sawyer_pick_place_v2.py

```diff
@@ -59,7 +59,7 @@ def __init__(self):
 
     @property
     def model_name(self):
-        return get_asset_full_path('sawyer_xyz/sawyer_pick_place_v2.xml')
+        return get_asset_full_path('sawyer_xyz/sawyer_pick_place_v2.xml', True)
```

Comment:
instead of using a boolean, please make a variant of this function `get_asset_path_v2`